### ISSUE TITLE
SSH to Private IP

### ISSUE THREAD

**Issue by tlongren** [pearkes/tugboat#169]
Hi!

I just installed Xubuntu 15.04 on my laptop, a new version of tugboat than is on my main workstation.

When I do 
```
tugboat ssh dropletname
```
It attempts to connect to the private (internal) IP address. I have done little research into it, but wanted to make a note here quick. It perhaps isn't related to tugboat and something with the droplets.

Not using tugboat, I can SSH to the external IP just fine.

Thoughts?

Thanks for tugboat, btw, it's an awesome tool.

**Comment by vmatekole** [comment]
Hi, I can also confirm I have same issue, with tugboat 1.0

**Comment by tlongren** [comment]
Definitely a somewhat recent issue. I have tugboat 0.2.8 installed on my main workstation and it connects to the external IP address of the specified droplet, as expected.

**Comment by PhilETaylor** [comment]
ah so this is what my problem is - it just hangs on connecting when I do a tugboat ssh ... so its trying to connect to the wrong IP  ?

**Comment by petems** [comment]
@tlongren So we allowed people to set if they wanted to connect to the private or public IP in 7b307b1e93ee2a90cbc550e6b7d059be9095fcc2, looks like that might've gone wrong.

Can you try adding the following into your ~/.tugboat file:

```yaml
use_public_ip: 'true'
```

And see if that fixes it?

**Comment by petems** [comment]
Can you also show the contents of your ~/.tugboat file (with the API keys redacted)

**Comment by PhilETaylor** [comment]
I'm using ```Tugboat 1.0.0``` installed with gem

my .tugboat file content:

    ---
    authentication:
      client_key: redacted
      api_key: redacted
    ssh:
      ssh_user: root
      ssh_key_path: "~/.ssh/id_rsa"
      ssh_port: '5819'
    defaults:
      region: '1'
      image: '350076'
      size: '66'
      ssh_key: id_rsa
      private_networking: 'false'
      backups_enabled: '1'
      use_public_ip: 'true'

When running command ```tugboat ssh 100``` I get:

    Phils-iMac:~ phil$ tugboat ssh 100
    Droplet fuzzy name provided. Finding droplet ID...done, 2266854 (lon100.myjoomla.com)
    Executing SSH (lon100.myjoomla.com)...
    ssh: connect to host 10.131.248.10 port 5819: Operation timed out

Note that ```10.131.248.10``` is the private IP address of the droplet with ```178.62.62.30``` as a public IP address.

**Comment by PhilETaylor** [comment]
Note that a command with option ```-e``` works and allows ssh to public IP address

This command works for me.
```tugboat ssh 100 -e```

**Comment by tlongren** [comment]
@petems, same deal here as @PhilETaylor.

```
tugboat ssh droplet -e
```
works like @PhilETaylor suggested.

**Comment by petems** [comment]
Weird, that should be the default. I'll see if I can fix up the logic.

**Comment by petems** [comment]
Fixed and released in 1.1.0

I didn't catch it because in my acceptance tests, because I don't have any servers with private IPs. Will update.
